spec.html

```diff
@@ -26358,6 +26485,7 @@ <h1>
               InnerModuleLoading (
                 _state_: a GraphLoadingState Record,
                 _module_: a Module Record,
+                _loadRecursively_: a Boolean
```

Comment:
We almost always prefer passing two-inhabitant enums over Booleans. They're more descriptive and they're less disruptive when we make a change.

Reply:
Done.